src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlCommandBuilderTest.cs

```diff
@@ -64,6 +67,8 @@ public void CatalogSeparator()
         {
             SqlCommandBuilder cb = new SqlCommandBuilder();
             Assert.Equal(".", cb.CatalogSeparator);
+            cb.CatalogSeparator = ".";
+            Assert.Equal(".", cb.CatalogSeparator);
```

Comment:
@lcheunglci if Assertion line 69 is correct, Do we need line 71?

Reply:
This is to add code coverage in the CatalogSeparator property that checks if a "." is set even though it does default to it.